{{cookiecutter.github_project_name}}/README.md

```diff
@@ -22,13 +17,9 @@ For a development installation (requires npm),
     $ pip install -e .
     $ jupyter nbextension install --py --symlink --sys-prefix {{ cookiecutter.python_package_name }}
     $ jupyter nbextension enable --py --sys-prefix {{ cookiecutter.python_package_name }}
-    $ jupyter labextension install js
-
-When actively developing your extension, build Jupyter Lab with the command:
-
-    $ jupyter lab --watch
 
-This takes a minute or so to get started, but then automatically rebuilds JupyterLab when your javascript changes.
+When actively developing your extension for JupyterLab, run the command:
 
-Note on first `jupyter lab --watch`, you may need to touch a file to get Jupyter Lab to open.
+    $ jupyter labextension develop {{ cookiecutter.python_package_name  }}
```

Comment:
Should it also mention the `--overwrite` flag?

Reply:
I think so: https://github.com/jupyter-widgets/widget-cookiecutter/pull/76